src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -804,15 +806,15 @@ public void execute() throws GitException, InterruptedException {
     public void clean(boolean cleanSubmodule) throws GitException, InterruptedException {
         reset(true);
 	String cmd = "-fdx";
-	if (cleanSubmodule) cmd = "-ffdx";
-
+    if (cleanSubmodule) cmd = "-ffdx";
+    
 	launchCommand("clean", cmd);
-    }
+}
 
-    /**
-     * Remove untracked files and directories, including files listed
-     * in the ignore rules.
-     *
+/**
```

Comment:
Remove this whitespace only change please.

Reply:
ditto